docs/src/adoc/index.adoc

```diff
@@ -1608,6 +1608,22 @@ int[] userIds = handle.createQuery("select user_ids from groups where id = :id")
       .findOnly();
 ----
 
+You can also use Collections in place of arrays, but you'll need to provide
+generic type information if you're using the fluent API:
+
+[source,java]
+----
+handle.createUpdate("insert into groups (id, user_ids) values (:id, :userIds)")
+      .bind("id", 1)
+      .bindByType("userIds", Arrays.asList(10, 5, 70), new GenericType<List<Integer>>() {})
```

Comment:
I was thinking we could add a `bindArray(column, collection, elementType)` method to make array binding simpler in core.

(in a separate PR)

Reply:
Yes, that would indeed be nice - I've created an issue for this https://github.com/jdbi/jdbi/issues/1274